### ISSUE TITLE
New "Execute" operation

### ISSUE THREAD

**Issue by VasasA** [backdrop-contrib/feeds#52]
A new "Execute" operation can be useful on the page of "Feeds importers". We can open the import pages with it. (I will create a PR.)
![kép](https://user-images.githubusercontent.com/8567234/84597192-93ec8200-ae62-11ea-8409-a0f5500d989d.png)

**Comment by herbdool** [comment]
Thanks for the idea and the PR @VasasA!

I am thinking we need to account for importers that are attached to content types as well. I first thought about how to change your link to account for that, but I noticed there's already an "Attached To" column with a link. I decided to consolidate them in that column (for now). Here's my alternative PR https://github.com/backdrop-contrib/feeds/pull/60. Let me know what you think.

![Screenshot from 2020-07-16 13-55-04](https://user-images.githubusercontent.com/1071483/87705706-4ec2b500-c76c-11ea-9a82-bbac17887530.png)
